app/assets/stylesheets/govuk_publishing_components/components/govspeak/_typography.scss

```diff
@@ -56,25 +56,25 @@
   }
 
   &.direction-rtl blockquote {
-    padding: 0 $gutter-two-thirds 0 0;
+    padding: 0 govuk-spacing(4) 0 0;
 
     @include govuk-media-query($from: desktop) {
-      margin: 0 (-$gutter) 0 0;
+      margin: 0 (- govuk-spacing(6)) 0 0;
```

Comment:
now i'm not sure which is correct `-(govuk-spacing(x)` or `- govuk-spacing(x)`

Reply:
They both work for single declarations, but on multiple declarations we have to isolate it from the previous value so it doesn't get computed.

Reply:
👍